.github/workflows/nightly.yml

```diff
@@ -0,0 +1,43 @@
+name: nightly
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build-and-publish-nightly:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install build
+      - name: Build the sdist
+        run: python -m build --sdist .
+        env:
+          BUILD_PYMC_NIGHTLY: true
```

Comment:
The one lingering thing I'm worried about is that someone could see this one yaml line and think if they change it to false the nightly build will stop, when in fact they wont because what's being checked in the script is the presence of an environmental var, not that its True or False. Any suggestions on how to mitigate this issue?

Reply:
I am unsure on this and it is beyond my experience with workflows. I copied this from https://github.com/aesara-devs/aesara/pull/656. Perhaps @dfm, who wrote the PR, might be able to shed some light on this?

Reply:
No problem, this as is is already a great contribution. This can be followed up later! :)